### DISCUSSION TITLE
Document.domain (deprecated)

### DISCUSSION THREAD

**Discussion by carlobarcena** [zloirock/core-js#1138] [General]
Document.domain is deprecated. Will you update this?
https://github.com/zloirock/core-js/blob/3051ab68b32548c178dce477141f20fb50e5d9d9/packages/core-js/internals/object-create.js#L59

Reference: https://developer.mozilla.org/en-US/docs/Web/API/Document/domain

**Comment by zloirock** [comment]
This code works only in ES3 engines, IE8- - it's a polyfill of `Object.create`.

Please, next time in such cases create issues instead of discussions.

**Reply by carlobarcena** [reply]
Ok, thank you.

**Reply by zloirock** [reply]
Also, the IE8- support will be removed in the next major release with this polyfill and this code.
